notes: add missing release notes for 6.1 (#5530)
concourse/concourse: release-notes/latest.md

  NOTE: And yes, you will need to log out and log back in after upgrading.

#### <sub><sup><a name="5371" href="#5371">:link:</a></sup></sub> fix, breaking

* Remove `Query` argument from `fly curl` command. 

  When passing curl options as `fly curl <url_path> -- <curl_options>`, the first curl option is parsed as query argument incorrectly, which then causes unexpected curl behaviour. #5366

  With fix in #5371, `<curl_options>` functions as documented and the way to add query params to `fly curl` is more intuitive as following:

  ```
  fly curl <url_path?query_params> -- <curl_options>
  ```

### <sub><sup><a name="5506" href="#5506">:link:</a></sup></sub> fix, breaking

> * When looking up credentials, we now prefer pipeline scoped credential managers over global ones. #5398

Reviewer: ```suggestion
* When looking up credentials, we now prefer pipeline scoped credential managers over global ones. #5506
```

https://github.com/concourse/concourse/pull/5398 is gzip stuff :grin:

#### <sub><sup><a name="5398" href="#5398">:link:</a></sup></sub> fix, breaking

* In a previous release, we made the switch to using `zstd` for compressing artifacts before they get streamed all over the place. This has proved to be unreliable for all our use cases so we switched the default back to `gzip`. We did make this configurable though so you can continue to use `zstd` if you so choose. #5398

#### <sub><sup><a name="5397" href="#5397">:link:</a></sup></sub> feature, breaking

> * @pnsantos updated the Material Design icon library version to `5.0.45`.

Reviewer: ```suggestion
* @pnsantos updated the Material Design icon library version to `5.0.45`. #5397
```

once formatted, makes navigating to the PR easier

>   **note:** some icons changed names (e.g. `mdi-github-circle` was changed to `mdi-github`) so after this update you might have to update some `icon:` references

Reviewer: ```suggestion
  **note:** some icons changed names (e.g. `mdi-github-circle` was changed to `mdi-github`) so after this update you might have to update some `icon:` references.
```

### <sub><sup><a name="5432" href="#5432">:link:</a></sup></sub> fix, breaking

* @tjhiggins updated the flag for configuring the interval at which concourse runs its internal components. `CONCOURSE_RUNNER_INTERVAL` -> `CONCOURSE_COMPONENT_RUNNER_INTERVAL`. #5432

#### <sub><sup><a name="5368" href="#5368">:link:</a></sup></sub> feature

> * Implemented the core functionality for archiving pipelines [RFC #33]. 

Reviewer: ```suggestion
* Implemented the core functionality for archiving pipelines [RFC #33].  #5368
```

  **note**: archived pipelines are neither visible in the web UI (#5370) nor in `fly pipelines`.

  **note:** archiving a pipeline will nullify the pipeline configuration. If for some reason you downgrade the version of Concourse, unpausing a pipeline that was previously archived will result in a broken pipeline. To fix that, set the pipeline again.

[RFC #33]: https://github.com/concourse/rfcs/pull/33

#### <sub><sup><a name="5459" href="#5459">:link:</a></sup></sub> feature

> * Since switching to using dex tokens, we started using the client credentials grant type to fetch tokens for the TSA. This seemed like a good opportunity to start bcrypting client secrets in the db. #5459

Reviewer: ```suggestion
* Since switching to using [dex](https://github.com/dexidp/dex) tokens, we started using the client credentials grant type to fetch tokens for the TSA. This seemed like a good opportunity to start `bcrypt`ing client secrets in the db. #5459
```

(just a nit - feel free to not accept :grin:)

### <sub><sup><a name="5519" href="#5519">:link:</a></sup></sub> fix

> * Thanks to some [outstanding debugging](https://github.com/concourse/concourse/issues/5385) from @agurney, we've fixed a deadlock in the notifications bus which caused the build page not to load under certain conditions.

Reviewer: ```suggestion
* Thanks to some [outstanding debugging](https://github.com/concourse/concourse/issues/5385) from @agurney, we've fixed a deadlock in the notifications bus which caused the build page not to load under certain conditions. #5519
```

### <sub><sup><a name="5091" href="#5091">:link:</a></sup></sub> feature

> * @evanchaoli added a global configuration to override the check interval for any resources that have been configured with a webhook token.

Reviewer: ```suggestion
* @evanchaoli added a global configuration to override the check interval for any resources that have been configured with a webhook token. #5091
```

#### <sub><sup><a name="5452" href="#5452">:link:</a></sup></sub> fix

* We fixed a bug where if you create a new build and then trigger a rerun build, both the builds will be stuck in pending state. #5452

#### <sub><sup><a name="5486" href="#5486">:link:</a></sup></sub> feature

> * We added a new flag to let you configure the garden network pool. 

Reviewer: ```suggestion
* We added a new flag (`CONCOURSE_CONTAINER_NETWORK_POOL`) to let you configure the network range used for allocating IPs for the containers created by Concourse. This is primarily intended to support the experimental [containerd](https://containerd.io) worker backend. Despite the introduction of this new flag, `CONCOURSE_GARDEN_NETWORK_POOL` is still functional for the (stable and default) Garden worker backend. #5486
```
